### ISSUE TITLE
Please add back the Wowza option

### ISSUE THREAD

**Issue by lizardpeter** [CoSMoSoftware/OBS-studio-webrtc#262]
Hello,

I finally got this software working flawlessly with Wowza, but when I went to update it I realized that the option was gone to stream to a Wowza server. Would you please be able to add this option back in for those of us who were finding it very useful?

Thank you,
Peter

**Comment by agouaillard-cosmo** [comment]
Hi peter.

We do not plan to bring it back nor maintain it in this fork in the near future as we are moving away from proprietary signalling for each vendor to embrace the new generic WHIP standard.
https://datatracker.ietf.org/doc/draft-murillo-whip/

You could leverage the Open-Source licence to have your own fork.

**Comment by lizardpeter** [comment]
Understandable. Will this WHIP standard be able to be used with Wowza?

**Comment by agouaillard-cosmo** [comment]
We hope that wowza will support it. Whether they do or not is their prerogative. They are supporting webrtc which is not a standard yet, so the hopes are high.

**Comment by lizardpeter** [comment]
Sorry to bother again, but I am trying to move my implementation from Wowza to Janus. Looking through the documentation and closed issues, I saw that you previously recommended people to try to connect WebRTC OBS to the Janus Plugin Demo for the Video Room. I am trying to do that using the url "wss://janus.conf.meetecho.com", the server room "1234", and the stream key "adminpwd". It is not working for me. Do you have any suggestions?

**Comment by agouaillard-cosmo** [comment]
Hi peter. No problem.

We agreed with lorenzo and his team to move support of Janus to WHIP. There were many reasons for that, but the simplest one is that Janus does not enable web socket signalling by default, and that we believe WHIP to be the future.

He wrote a very detailed blog post about it here:
https://www.meetecho.com/blog/whip-janus/

I'm not sure where the corresponding code is, but an email on the janus mailing list or to lorenzo directly should get you that information.

**Comment by lizardpeter** [comment]
Thank you. I will look into using WHIP. However, shouldn’t the old way of using secure web sockets also be working for me? I’ve tried wss://myDomainHere:portHere using the test room 1234. The issue is that OBS is saying I haven’t filled out all of the fields correctly if I don’t include a Stream Key. I thought this Stream Key wouldn’t be required if the room was not password protected. In all of the tutorials I can find, this Stream Key field was not in that build of OBS.

**Comment by lizardpeter** [comment]
I have tried again and have made sure that Janus is in fact working over secure websockets. I used the Janus Video Room Demo and hosted it on my site and connected it to my server using wss and was able to add many clients to it from Chrome. 

However, I am still unable to get it to work on OBS. Using the "Janus WebRTC Server" option does not work with any combination of inputs I could think of. The log is showing an error which I have attached below.

PEER CONNECTION CREATED
20:00:27.401: (openssl_identity.cc:43): Making key pair
20:00:27.401: (openssl_identity.cc:43): Making key pair
20:00:27.401: 
20:00:27.401: 
20:00:27.401: (openssl_identity.cc:91): Returning key pair
20:00:27.401: 
20:00:27.401: (openssl_certificate.cc:59): Making certificate for WebRTC
20:00:27.401: 
20:00:27.401: (peer_connection.cc:1829): Adding audio transceiver in response to a call to AddTransceiver.
20:00:27.401: 
20:00:27.401: (peer_connection.cc:1829): Adding video transceiver in response to a call to AddTransceiver.
20:00:27.401: 
20:00:27.401: (openssl_certificate.cc:109): Returning certificate
20:00:27.401: 
20:00:27.401: (webrtc_session_description_factory.cc:481): Setting new certificate.
20:00:27.401: 
20:00:27.401: Server Room:      1234
20:00:27.401: Stream Key:       lizardpeter
20:00:27.401: 
20:00:27.401: CONNECTING TO wss://xeniea.com:8989
20:00:27.409: > set_fail_handler > handleFail called
20:00:27.409: Connection Timed out
20:00:27.409: WebRTCStream::onDisconnected
20:00:27.410: (peer_connection.cc:4794): Session: 8660296620586414921 Old state: kStable New state: kClosed
20:00:27.410: 
20:00:27.410: WebRTCStream::OnIceConnectionChange [6]
20:00:27.410: WebRTCStream::OnConnectionChange [5]
20:00:27.410: (paced_sender.cc:176): ProcessThreadAttached 0x0
20:00:27.410: 
20:00:27.410: (paced_sender.cc:176): ProcessThreadAttached 0x0
20:00:27.410: 
20:00:27.410: Sending destroy message...
20:00:27.410: Error sending destroy message
20:00:27.410: Sending destroy message...
20:00:27.411: Error sending destroy message
20:00:27.411: Output 'adv_stream':  Reconnecting in 10 seconds..
